### PR TITLE
Enable armv8 CI

### PR DESCRIPTION
**Pull request by digantdesai**

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -757,7 +757,7 @@ def get_git_commit_hash(length=8):
             "pytest-forked",
             "pytest-xdist",
             "scipy>=1.7.1",
-            "llnl-hatchet",
+            # "llnl-hatchet", # TODO: Re-enable this, not available on macos-arm64
```
Comment (minjang) [inline python/setup.py:760]: Will it be okay in other platforms? It's okay as you marked as TODO:

**Review by minjang — APPROVED** [review]
Let's try!

**Comment by digantdesai** [comment]
Ah still failing, let me fix it.

File: third_party/cpu/backend/driver.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 from triton.backends.driver import DriverBase
 from triton.backends.compiler import GPUTarget
 
+from pathlib import Path
```
Comment (int3) [inline third_party/cpu/backend/driver.py:15]: hm is this import getting used?

**Pull request merged**